adam-cli/src/test/scala/org/bdgenomics/adam/cli/ADAM2FastaSuite.scala

```diff
@@ -17,18 +17,16 @@
  */
 package org.bdgenomics.adam.cli
 
-import com.google.common.io.Files
 import java.io.File
-import org.apache.spark.rdd.RDD
-import org.bdgenomics.adam.rdd.ADAMContext._
+
```

Comment:
I admit to opening up IntelliJ only rarely, and am likely the culprit for some of those unused imports.  If you have settings that work well with our style, maybe we could add them to the contributing doc.